doc/sheet_functions.md

```diff
@@ -6,6 +6,9 @@
 Active Addresses&quot; refers to the number of unique addresses that
 participated in transactions on a blockchain.</p>
 </dd>
+<dt><a href="#SAN_ACTIVE_ADDRESSES_24H">SAN_ACTIVE_ADDRESSES_24H(projectSlug, from, to)</a> ⇒ <code>Array</code></dt>
```

Comment:
Do we need `SAN_ACTIVE_ADDRESSES_24H`? Isn't `SAN_ACTIVE_ADDRESSES` with interval argument enough?

Reply:
https://github.com/santiment/sanbase2/blob/master/lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json#L157

https://github.com/santiment/sanbase2/blob/master/lib/sanbase/clickhouse/metric/metric_files/available_v2_metrics.json#L174

They're 2 different metrics, this PR is motivated by the fact that ``SAN_ACTIVE_ADDRESSES_24H`` is the same as ``SAN_ACTIVE_ADDRESSES`` when the interval is ``1d``

Reply:
Actually, you're right, going to look into it